CHANGES.md

```diff
@@ -116,6 +116,7 @@
 
 ## Bugfixes
 
+* Fixed numpy operators in ApproximateQuantiles (Python) ([BEAM-9579](https://issues.apache.org/jira/browse/BEAM-9579)).
```

Comment:
@amaliujia - Should we wrap up the 2.20 release section and start a new section. This is the first release we are trying out changes.md, so we may need to figure out a good point where we need to update this file.

Maybe we can do it after release branch cut. What do you think?

Reply:
Yep. Release branch is cut is a good cut point. Later when if there is blocker JIRA that adds changes to release branch, release manager can ask to add something to `changes.md`.